src/dnx.host/RuntimeBootstrapper.cs

```diff
@@ -64,6 +64,8 @@ public static Task<int> ExecuteAsync(string[] args, FrameworkName targetFramewor
 
             // These options were handled in the native code, but got passed through here.
             // We just need to capture them and clean them up.
+            var optionProject = app.Option("--project|-p", "Path to the project.json file or the application folder. Defaults to the current folder if not provided.",
+                CommandOptionType.NoValue);
```

Comment:
I found the option `CommandOptionType.NoValue` misleading. Will change it to `SingleValue` make this option mandatory?